haas/rest.py

```diff
@@ -164,10 +170,13 @@ def foo(bar, baz, quux):
           whose second is an integer (the status code).
     """
     def register(f):
+
+        methods = method if isinstance(method, list) else [method]
```

Comment:
I think taking a list from everyone would be ok too, except it would be adding characters to everyone whereas this functionality is mostly useful for testing.

There are a number of API calls that differ only in method (like show_network() and network_delete()), though the benefits of having them clearly defined probably outweigh the combining of the common code (which would most likely be auth and fetching the accompanying objects).